test/abap/pretty_printer.ts

```diff
@@ -3,6 +3,17 @@ import {PrettyPrinter} from "../../src/abap/pretty_printer";
 import {MemoryFile} from "../../src/files";
 import {Registry} from "../../src/registry";
 
+const testTitle = (text: string): string => { return text.split("\n")[0]; };
+const createRunner = (test: any, methodName: string): (() => void) => {
```

Comment:
this is a function?

Reply:
only used in one place

Reply:
Yes, but the proposal is to apply to all suites in this test - the code is almost same. But have a look below comment - it is all connected.

Reply:
removed